DetectorDescription/DDCMS/src/DDFilteredView.cc

```diff
@@ -13,6 +13,19 @@ using namespace edm;
 using namespace std;
 using namespace cms::dd;
 
+dd4hep::Solid DDSolid::solidA() const {
+  if (dd4hep::instanceOf<dd4hep::SubtractionSolid>(solid_) || dd4hep::instanceOf<dd4hep::UnionSolid>(solid_) ||
+      dd4hep::instanceOf<dd4hep::IntersectionSolid>(solid_)) {
+    const TGeoCompositeShape* sh = (const TGeoCompositeShape*)solid_.ptr();
+    const TGeoBoolNode* boolean = sh->GetBoolNode();
+    TGeoShape* solidA = boolean->GetLeftShape();
```

Comment:
Seems a bit confusing to re-use the name of the member function within that function. How about instead:
```
TGeoShape* solidLeft = boolean->GetLeftShape();
```

Reply:
I would naively agree with @cvuosalo comment, although the original code is strictly speaking not wrong

Reply:
I think it's a question of a style. I find it more clear that the method that requests ```solidA()``` declares that the ```solidA``` is a left shape of a boolean.